### PR TITLE
Add a playground logo

### PR DESCRIPTION
**Pull request by msm-code**
Co-authored-by: Nsty <nsty@cert.pl>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Karton Playground: getting started
+# Karton Playground
+
+![](./img/logo-sm.png)
```
Comment (chivay) [inline README.md]: Maybe use `width` property instead of uploading multiple files to the repo? :thinking:

Comment (chivay) [reply README.md]: example: 
``

Comment (msm-code) [reply README.md]: I don't even know if we want the big or small one :thinking: but right

Comment (msm-code) [reply README.md]: Changed to full-size logo (I don't know how big it should be :thinking: )

**Review by chivay — APPROVED** [review]

**Review by BonusPlay — APPROVED** [review]

**Pull request merged**